### DISCUSSION TITLE
Does anyone still need an SWF graphics export?

### DISCUSSION THREAD

**Discussion by codemanyak** [fesch/Structorizer.Desktop#1173] [Polls]
Structorizer has been providing a picture export to SWF (Shockwave Flash) format, e.g. via menu entry "File > Export > Picture > SWF". Flash support by Adobe and most browsers expired, though, many years ago. Hence it doesn't seem to make sense to maintain this Flash export function any longer, though there may still be some Flash plugins available (e.g. for Chrome). So I wonder whether there might be objections to kick it out.

**Comment by fesch** [comment]
I have no objections to remove it ...

**Reply by codemanyak** [reply]
Though being pretty sure that no-one will protest, I just put deprecation hints to the respective menu item captions and into the User Guide for now. The version after the one on the brink will then certainly remove the SWF export.

**Comment by codemanyak** [comment]
From version 3.32-24 on, Structorizer will no longer offer SWF export.
